src/app.ts

```diff
@@ -1,15 +1,18 @@
 import express, { Request, Response, NextFunction } from "express";
-import todoRoutes from "./routes/todos";
 
-const app = express();
+const port = 3000;
```

Comment:
@zoheirkhonyagar  get port from env variable

Reply:
you should add it on another pull request